velox/connectors/hive/storage_adapters/hdfs/tests/HdfsFileSystemTest.cpp

```diff
@@ -350,11 +361,45 @@ TEST_F(HdfsFileSystemTest, multipleThreadsWithFileSystem) {
   }
 }
 
+TEST_F(HdfsFileSystemTest, write) {
+  std::string path = "/a.txt";
+  auto writeFile = openFileForWrite(path);
+  std::string data = "abcdefghijk";
+  writeFile->append(data);
+  writeFile->append(data);
+  writeFile->append(data);
+  writeFile->flush();
```

Comment:
Could you also add the case the appends are interleaved with flush?

Reply:
done